engine/core/IgeEntity.ts

```diff
@@ -2669,6 +2667,34 @@ export class IgeEntity extends IgeObject implements IgeCanRegisterById, IgeCanRe
 		return this;
 	}
 
+	tileX () {
+		if (this._parent && this._parent._tileWidth !== undefined) {
+			return Math.floor(this._translate.x / this._parent._tileWidth);
+		}
+	}
+
+	tileY () {
+		if (this._parent && this._parent._tileHeight !== undefined) {
+			return Math.floor(this._translate.y / this._parent._tileHeight);
+		}
+	}
+
+
+	tileZ (): number;
+
+	tileZ (val: number): this;
+
+	tileZ (val?: number) {
+		if (this._parent && val !== undefined) {
```

Comment:
Why is tileZ different from the others? Should we support setters for the other two as well?